### DISCUSSION TITLE
Why is greylisting skipped

### DISCUSSION THREAD

**Discussion by mschering** [rspamd/rspamd#4983] [Q&A] [answered]
Hi,

I am using the default setup for greylisting. But I'm seeing a lot of these in my log:

```
2024-05-24 08:51:15 #2371710(rspamd_proxy) <f78dcf>; lua; greylist.lua:331: Score too low - skip greylisting
```

I found the setting "greylist_min_score" but it should be unset by default. Am I wrong to assume it should be greylisting any score?

I tried to set:

```
greylist_min_score = 0.1;
```

But it makes no difference. I still see a lot of those messages.

I'd like to enable it for everybody.

Best regards,
Merijn

**Comment by tobiaskolbe** [comment, marked as answer]
Is it possible that you have set a different minimal score in another place?
According to `modules.d/greylist.conf` on my system rspamd will include different files:
```
.include(try=true,priority=5) "${DBDIR}/dynamic/greylist.conf"
.include(try=true,priority=1,duplicate=merge) "$LOCAL_CONFDIR/local.d/greylist.conf"
.include(try=true,priority=10) "$LOCAL_CONFDIR/override.d/greylist.conf"
```
The dynamic configuration from the web UI may override your configuration because of it's priority:
<img width="657" alt="rspamd-webui" src="https://github.com/rspamd/rspamd/assets/107499496/4b7607de-7f91-4a1e-99da-bed081c3ec00">

**Reply by mschering** [reply]
Sorry for my ignorance. It was the webgui setting indeed. I checked all the files in the $LOCAL_CONFDIR but didn't realise there was another location.

Thanks for your help!
